### ISSUE TITLE
upgrading point cloud from master branch to 0.2.0

### ISSUE THREAD

**Issue by SignpostMarv** [vizicities/vizicities#156]
This chunk of code is used to populate the vertices of a THREE.ParticleSystem (named THREE.PointCloud in the latest version of three.js), which renders the geolocation of various social media icons on the map. With the master branch code, it took some digging but I figured out how to add particles to my map (see issue #155). Having a little bit of trouble performing the same task with the 0.2.0 branch- any help would be appreciated :)

```javascript
define(function(){ return function(options){
    var
        coords = options.city.geo.projection([
            parseFloat(options.longitude),
            parseFloat(options.latitude)
        ])
    ;
    return (
        options.geometry.vertices.push(new THREE.Vector3(
            coords[0],
            options.altitude * options.city.geo.pixelsPerMeter,
            coords[1]
        )) - 1
    );
};});
```

**Comment by SignpostMarv** [comment]
guessing this'd be something suitable for a blueprint wotsit, rather than fudging it directly into three.js?

**Comment by SignpostMarv** [comment]
cough. any assistance ?

**Comment by antont** [comment]
yes the idea of 0.2.0 is that you can do whatever data inputs & visualizations cleanly as plugins, and those are called blueprints. input bluprints and output blueprints. no need to hack vizi core (for example the scene creation code9 like in 0.1.

this is an example of such an output plugin that draws points:
https://github.com/vizicities/vizicities/blob/0.2.0/src/Blueprint/BlueprintOutputDebugPoints.js
